card/lib/generators/card/USAGE

```diff
@@ -0,0 +1,21 @@
+Description
+
+Examples
+  wagn generate card new cards pre_fix
```

Comment:
> I'm not sure you quite understand that there really isn't much to decko-rails

I'm very clear on this and have been saying this all along.  Very little. It should be a very lightweight gem.  You've put a lot more in there than I would want there.  NO core decko code should be there.

> Both cases are a rails app, but in one you don't open up the app at all. Extensions are modules within the app.

Wagn happens to be a rails app now.  For the purposes of this architectural decision, it's better to pretend it's not.